.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,6 +23,7 @@ https://prairielearn.readthedocs.io/en/latest/contributing
 
 - Share how you tested your changes.
 - If you're fixing a bug, explain how to reproduce the original issue.
+- If possible, include the commit that testing was performed on.
```

Comment:
Is this a helpful recommendation? This will almost immediately cease to be useful information once one starts iterating in response to feedback. Let's discuss tomorrow if needed.

Reply:
Changing this to note chronological testing comments instead.